src/app/pages/home/components/task-calendar/task-calendar.component.html

```diff
@@ -48,17 +31,23 @@
             id="task-item"
             (tap)="clicked(task)"
             *ngIf="task.showInCalendar"
+            [class.task-item-name]="isTaskNameShown"
           >
             <ion-row>
               <ion-col>
                 <div class="task-info-left">
-                  <div class="start-time">{{ getStartTime(task) }}</div>
+                  <div *ngIf="isTaskNameShown" class="start-time">
+                    {{ task.name }}
+                  </div>
+                  <div *ngIf="!isTaskNameShown" class="start-time">
+                    {{ getStartTime(task) }}
+                  </div>
```

Comment:
I was wondering if we should have subcomponents that are inserted based on the configured values instead of if/else control loops (as it can get messy when we have a large codebase). WDYT?

Reply:
Ah yes do you mean like creating a separate subcomponent for a task calendar row (which contains the name, est. time, etc)?

Reply:
Yes i think that would work (so extensible in the future as well if we need to add new variations of this as I am sure some users would want to add/remove estimate time or number of questions or something else)